README.md

```diff
@@ -23,6 +23,17 @@ When the program runs, it should create and show a list of ten numbered menu ite
 10. creamy taco cake
 ```
 
+## Test & Verify
+
+Before you submit your work it's important to test your program and ensure it's working properly.  Later we will teach you nifty ways to automate this testing, but for now we'll do things the old fashioned way.
+
+Make sure you run your program and ensure it's working.  Among your tests ensure that:
+
+*  The menu items are selected randomly.
+*  There are 10 menu items.
```

Comment:
Would be good to add "numbered 1 through 10" since they often h ave trouble with that due to the zero-indexing